.github/workflows/ci.yml

```diff
@@ -51,3 +51,141 @@ jobs:
+          key: ${{ runner.os }}-cargo-registry-cache-${{ matrix.rust }}-${{ hashFiles('**/Cargo.lock') }}
+          restore-key: |
+            ${{ runner.os }}-cargo-registry-cache-${{ matrix.rust }}-
+            ${{ runner.os }}-cargo-registry-cache-
+
+      - name: Cache cargo registry index
+        uses: actions/cache@v1
+        with:
+          path: ~/.cargo/registry/index
+          key: ${{ runner.os }}-cargo-registry-index-${{ matrix.rust }}-${{ hashFiles('**/Cargo.lock') }}
+          restore-key: |
+            ${{ runner.os }}-cargo-registry-index-${{ matrix.rust }}-
+            ${{ runner.os }}-cargo-registry-index-
+
+      - name: Cache cargo git db
+        uses: actions/cache@v1
+        with:
+          path: ~/.cargo/git/db
+          key: ${{ runner.os }}-cargo-git-db-${{ matrix.rust }}-${{ hashFiles('**/Cargo.lock') }}
+          restore-key: |
+            ${{ runner.os }}-cargo-git-db-${{ matrix.rust }}-
+            ${{ runner.os }}-cargo-git-db-
+
+      - name: Cache cargo build
+        uses: actions/cache@v1
+        with:
+          path: target
+          key: ${{ runner.os }}-cargo-build-target-${{ matrix.rust }}-${{ hashFiles('**/Cargo.lock') }}
+          restore-key: |
+            ${{ runner.os }}-cargo-build-target-${{ matrix.rust }}-
+            ${{ runner.os }}-cargo-build-target-
+
+      - name: Install ${{ matrix.rust }} Rust
+        run: |
+          if [[ ! -d "$HOME/.cargo" || ! -d "$HOME/.rustup" ]]; then
+              curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup-init.sh
+              sh rustup-init.sh -y --default-toolchain none
+          fi
+          rustup set profile minimal
+          rustup update ${{ matrix.rust }}
+          rustup default ${{ matrix.rust }}
+
+      - name: Install lint tools
+        if: matrix.rust == 'stable'
+        run: |
+          rustup component add rustfmt
+          rustup component add clippy
+
+      - name: Cargo clean on new rustc version
+        run: script/ci/cargo-clean-on-new-rustc-version.sh
+
+      - name: Install required system libraries
+        run: |
+          sudo tee /etc/apt/sources.list.d/pgdg.list <<END
+          deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main
+          END
+          wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+          sudo apt-get update
+          sudo apt-get install -y libpq-dev postgresql-11
+          mkdir -p /etc/postgresql/11/main
+          sudo tee /etc/postgresql/11/main/pg_hba.conf <<END
+          local   all             all                                     trust
+          host    all             all             127.0.0.1/32            trust
+          END
+          sudo systemctl restart postgresql@11-main
+
+      - name: Setup database
+        run: |
+          which diesel || cargo install diesel_cli --force --vers $(cat .diesel_version) --no-default-features --features postgres
```

Comment:
I don't think we need `--force` here anymore because we'll only install `diesel_cli` if it's not present? I don't think it's hurting anything either though... [we removed it from the travis config recently](https://github.com/rust-lang/crates.io/commit/f1afb8f95b563f2b47793158ec0d7202096bb03e#diff-354f30a63fb0907d4ad57269548329e3). What do you think?